spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -414,5 +414,13 @@
         RUBY
       end
     end
+
+    context 'with `join` method with implicit receiver' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY)
+          join(Rails.root, path)
+        RUBY
+      end
+    end
```

Comment:
Test and implementation like this seem to be required for both `EnforcedStyle: 'arguments'` and `EnforcedStyle: 'slashes'`.

Reply:
Good point, fixed.